### PR TITLE
test(e2e): check-xpub-VTC

### PR DESCRIPTION
**Pull request by bosomt**
check presence of VTC xpub in account detail.

### CODE REVIEW

**Comment by szymonlesisz** [comment]
please change the commit message (see QA slack thread)

File: packages/integration-tests/projects/suite-web/tests/wallet/check-xpub-vtc.test.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// @group:suite
+// @retry=2
+
+describe('Check Vertcoin XPUB', () => {
+    beforeEach(() => {
+        cy.task('startEmu', { wipe: true });
+        cy.task('setupEmu', {
+            needs_backup: false,
+            mnemonic: 'all all all all all all all all all all all all',
```
Comment (mroz22) [inline packages/integration-tests/projects/suite-web/tests/wallet/check-xpub-vtc.test.ts:9]: performacne nitpick: might be better to use another seed, where discovery is faster

File: packages/integration-tests/projects/suite-web/tests/wallet/check-xpub-vtc.test.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        cy.getTestElement('@settings/wallet/network/vtc').click({ force: true });
+        //
+        // Test execution
+        //
+        cy.getTestElement('@suite/menu/wallet-index').click();
+        cy.discoveryShouldFinish();
+        cy.getTestElement('@account-menu/vtc/normal/0/label').click();
+        cy.getTestElement('@wallet/menu/wallet-details').click();
+        cy.getTestElement('@wallets/details/show-xpub-button').click();
+        cy.getTestElement('@xpub-modal/xpub-field').should('exist');
```
Comment (mroz22) [inline packages/integration-tests/projects/suite-web/tests/wallet/check-xpub-vtc.test.ts:46]: nitpick: maybe I would also check xpub value itself here. Or you may also create a snapshot for this modal window.

File: packages/integration-tests/projects/suite-web/tests/wallet/check-xpub-vtc.test.ts (hunk 1 above)
Comment (bosomt) [reply packages/integration-tests/projects/suite-web/tests/wallet/check-xpub-vtc.test.ts:9]: we will consider it for next versions of tests
1) seed with less transactions
2) checking xpub value
3) checking all xpubs for all coins in one test to optimize test performance
now are in process of learning :)

**Review by ondracja — APPROVED** [review]

**Pull request merged**